### ISSUE TITLE
Register new globus endpoint

### ISSUE THREAD

**Issue by bess** [pulibrary/pdc_discovery#304]
Register the new globus endpoint with OIT and document the process in the RDSS handbook.

Goal: Someone other than Francis needs to be able to do this going forward.

- [ ] for staging
- [ ] for production

Document:
* the process so far
* contacts at OIT
* Document here: https://github.com/pulibrary/rdss-handbook/blob/main/globus.md

**Comment by bess** [comment]
Re-assigning this to @carolyncole as discussed at all-hands meeting.

**Comment by carolyncole** [comment]
Waiting on support ticket.  I can no longer connect AWS S3 to our collections in globus.  This was working on Monday, but stopped working yesterday.  I created a support ticket in globus: https://support.globus.org/hc/en-us/requests/357764

Currently the PreCuration Staging Endpoint is the only one still connected to S3.  I tried to reregister the PostCuration Endpoint and it failed, even though it had worked on Monday.  I though maybe the exposure yesterday may have caused the issue so I created a new key, but that also did not help.
